ui/perfherder/graphs/GraphsViewControls.jsx

```diff
@@ -113,38 +113,40 @@ export default class GraphsViewControls extends React.Component {
               />
             )}
 
-            <Row className="justify-content-start pt-2 pb-5 max-width-default mx-auto">
-              {highlightedRevisions.length > 0 &&
-                highlightedRevisions.map((revision, index) => (
-                  // eslint-disable-next-line react/no-array-index-key
-                  <Col sm="2" className="pl-0 pr-3" key={index}>
-                    <Input
-                      type="text"
-                      name={`revision ${revision}`}
-                      placeholder="revision to highlight"
-                      value={revision}
-                      onChange={event =>
-                        this.changeHighlightedRevision(
-                          index,
-                          event.target.value,
-                        )
-                      }
-                    />
-                  </Col>
-                ))}
-              <Col sm="auto" className="pl-0">
-                <Button
-                  color="darker-info"
-                  outline
-                  onClick={() =>
-                    updateStateParams({ highlightAlerts: !highlightAlerts })
-                  }
-                  active={highlightAlerts}
-                >
-                  Highlight alerts
-                </Button>
-              </Col>
-            </Row>
+            {!showTable && (
```

Comment:
So we just want to hide the "highlight alerts" button in the table view until that logic is fixed.